src/CoreBundle/private/library/classes/components/imageMagick/imageMagick.class.php

```diff
@@ -519,16 +493,90 @@ public function ResizeImage($iWidth, $iHeight)
         return $returnVal;
     }
 
+    protected function getImageOrientation(): ?string
+    {
+        $exifData = $this->getExifData();
+
+        return $exifData['Orientation'] ?? null;
+    }
+    
+    protected function getRotationCommandForOrientation(string $orientation): ?string
+    {
+        $command = null;
+        
+        switch ($orientation) {
+            case '1':
+            case 'TopLeft':
+                break;
+            case '2':
+            case 'TopRight':
+                $command = "-flop ";
```

Comment:
heißt das wirklich flop und nicht auch flip? und falls ja, dann könnte man das mit dem "flip-block" zusammenführen

Reply:
yes. it is really flop.
"creates a horizontal mirror image by reflecting the pixels around the central y-axis"